wiki/Guides/Compressing_files/fr.md

```diff
@@ -0,0 +1,99 @@
+# Compression des fichiers
+
+Chaque beatmap a une limite de taille de fichier dictée par sa longueur totale. Ce guide vous aidera à ramener votre beatmap sous cette limite.
+
+Il existe 2 types de compression. La compression **avec perte** et la compression **sans perte**.
+
+- **Sans perte** implique que la qualité ne se dégrade jamais et peut être compressée ou décompressée à plusieurs reprises.
+- **Avec perte** est une forme de compression beaucoup plus puissante qui sacrifie la qualité à la demande d'espace/de processeur.
+
+## Vidéo
+
+Les vidéos sont généralement l'endroit où les gens se trompent. Il existe des milliers de codecs vidéo, et *croyez-le ou non, YouTube est généralement un endroit terrible pour obtenir des vidéos*.
+
+Le H.264 est un bon codec à utiliser, mais comme la plupart des codecs vidéo, il est avec perte. Cela signifie que vous devez éviter de ré-encoder la vidéo plusieurs fois, mais plutôt encoder une fois à partir de la source haute résolution.
+
+### Utilisation de Handbrake
+
+Cette section vous montre comment supprimer le son des vidéos en utilisant [Handbrake](https://handbrake.fr/) pour réduire la taille du fichier vidéo.
+
+1. Ouvrez Handbrake et importez votre fichier vidéo. Vous pouvez glisser et déposer votre fichier dans Handbrake ou l'importer manuellement en cliquant sur l'option `Fichier`.
+
+![Importation de fichiers vidéo dans Handbrake](img/import-handbrake-FR.png "Importation de la vidéo dans Handbrake")
+
+2. Sélectionnez le préréglage `Fast 720p30`.
+
+![Sélection du préréglage vidéo](img/preset-handbrake-FR.png "Sélection du préréglage")
+
+3. Sélectionnez l'onglet `Audio` et supprimez toutes les pistes audio. Faites de même pour les sous-titres en allant dans l'onglet `Sous-titres` et en supprimant toutes les entrées.
+
+![Suppression des pistes audio de l'onglet audio dans Handbrake](img/removeaudio-handbrake-FR.png "Suppression des pistes audio")
```

Comment:
is the reason you've not put any arrows that your tool doesn't have a way to put them in?

![image](https://user-images.githubusercontent.com/36758269/145277559-59448243-3c18-4409-b0d1-091c60cfcc3a.png)